include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -181,9 +190,10 @@ SECTIONS
 
     GROUP_START(RAMABLE_REGION)
 
-#if defined(CONFIG_SOC_SERIES_STM32F0X) && !defined(CONFIG_IS_BOOTLOADER)
+#if defined (CONFIG_SOC_SERIES_NRF51X) ||	\
+	defined(CONFIG_SOC_SERIES_STM32F0X) && !defined(CONFIG_IS_BOOTLOADER)
     /* Must be first in ramable region */
-	SECTION_PROLOGUE(.st_stm32f0x_vt,(NOLOAD),)
+	SECTION_PROLOGUE(.fixed_vector_table,(NOLOAD),)
```

Comment:
The name .fixed_vector_table maybe misleading. It is not fixed in the sense of const. Maybe call it .ram_vector_table.